Description: "template inline" instead of "inline template"
  Fix FTBFS of ompl against boost1.58 by gcc5.
Author: v4hn <me@v4hn.de>
Origin: https://github.com/v4hn/ublas/commit/78eecc8bbf035c01316647839be4094748f8e77d
      https://svn.boost.org/trac/boost/attachment/ticket/11207/patch_numeric-ublas-storage.hpp.diff
Bug-Debian: https://bugs.debian.org/797281
Bug-Debian: https://bugs.debian.org/799819
Reviewed-By: Anton Gladky <gladk@debian.org>
Last-Update: 2015-10-03

--- boost1.58-1.58.0+dfsg.orig/boost/numeric/ublas/storage.hpp
+++ boost1.58-1.58.0+dfsg/boost/numeric/ublas/storage.hpp
@@ -777,8 +777,8 @@ namespace boost { namespace numeric { na
         BOOST_UBLAS_INLINE
         shallow_array_adaptor (size_type size, pointer data):
             size_ (size), own_ (false), data_ (data, leaker<value_type> ()) {}
-        BOOST_UBLAS_INLINE
         template <size_t N>
+        BOOST_UBLAS_INLINE
         shallow_array_adaptor (T (&data)[N]):
             size_ (N), own_ (false), data_ (data, leaker<value_type> ()) {}
 
@@ -833,13 +833,13 @@ namespace boost { namespace numeric { na
         void resize (size_type size, pointer data, value_type init) {
             resize_internal (size, data, init, true);
         }
-        BOOST_UBLAS_INLINE
         template <size_t N>
+        BOOST_UBLAS_INLINE
         void resize (T (&data)[N]) {
             resize_internal (N, data, value_type (), false);
         }
-        BOOST_UBLAS_INLINE
         template <size_t N>
+        BOOST_UBLAS_INLINE
         void resize (T (&data)[N], value_type init) {
             resize_internal (N, data, init, true);
         }
